src/interaction/TabulatedAngular.py

```diff
@@ -82,6 +85,29 @@ def setPotential(self, type1, type2, potential):
         if not (pmi._PMIComm and pmi._PMIComm.isActive()) or pmi._MPIcomm.rank in pmi._PMIComm.getMPIcpugroup():
             self.cxxclass.setPotential(self, type1, type2, potential)
 
+
+class FixedTripleListTypesTabulatedAngularLocal(InteractionLocal, interaction_FixedTripleListTypesTabulatedAngular):
+    def __init__(self, system, vl):
+        if pmi.workerIsActive():
+            cxxinit(self, interaction_FixedTripleListTypesTabulatedAngular, system, vl)
+
+    def setPotential(self, type1, type2, type3, potential):
+        if pmi.workerIsActive():
+            self.cxxclass.setPotential(self, type1, type2, type3, potential)
+
+    def getPotential(self, type1, type2, type3):
+        if pmi.workerIsActive():
+            return self.cxxclass.getPotential(self, type1, type2, type3)
+
+    def setFixedPairList(self, fixedpairlist):
+        if pmi.workerIsActive():
+            self.cxxclass.setFixedPairList(self, fixedpairlist)
```

Comment:
is this supposed to be setFixedTripleList? (and same for other occurences of FixedPair below)